index.html

```diff
@@ -10,12 +10,13 @@
   <header>
     <nav>
       <label class="logo">Sahar</label>
-      <span><img src="img/Enabled.png" alt="burger" class="burger"></span>
-      <ul>
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
+      <ul class="sidebar" id="expend">
+        <li><a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a></li>
+        <li><a href="#portfolio" onclick="closeNav()">Portfolio</a></li>
+        <li><a href="#about" onclick="closeNav()">About</a></li>
+        <li><a href="#contact" onclick="closeNav()">Contact</a></li>
```

Comment:
#### OPTIONAL

- [ ] Kindly increase the font size of each of the mobile nav links